### ISSUE TITLE
Feature request: add a warning if runTaintAnalysis has been set in the psalm configuration file

### ISSUE THREAD

**Issue by MetalArend** [vimeo/psalm#8365]
If the `runTaintAnalysis` configuration is being set in the `psalm.xml` configuration, it is probably an error, as it will make the rest of the whole configuration file obsolete. Could you add a warning in the output whenever that option is being set in the configuration, and not by a command line flag? Or could the output reference that there might be some unreported errors, because only the taint analysis has run?

**Comment by orklah** [comment]
Yeah, I guess we could make sure to explain which mode we're in (taint or analysis) in the message displayed at the beggining.

I don't want to remove the option because it allows using a single command line with just changing config file in order to run in different modes so it could break workflows

Would you be up for submitting a PR doing that? :)
